app/models/driver.rb

```diff
@@ -1,7 +1,8 @@
+#
 class Driver < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :database_authenticatable, :rememberable, :trackable, authentication_keys: [:phone]
```

Comment:
Why do you remove `:rememberable` from Driver's model in a request about adding the gem?